src/services/breakoutRoomsService.js

```diff
@@ -25,12 +25,12 @@ import { generateOcsUrl } from '@nextcloud/router'
 /**
  * Create breakout rooms for a given conversation
  *
- * @param token The conversation token
- * @param mode Either manual, auto, or free, see constants file
- * @param amount The amount of breakout rooms to be created
- * @param attendeeMap A json encoded Map of attendeeId => room number (0 based)
+ * @param {string} token The conversation token
+ * @param {string} mode Either manual, auto, or free, see constants file
+ * @param {number} amount The amount of breakout rooms to be created
+ * @param {string} attendeeMap A json encoded Map of attendeeId => room number (0 based)
  * (Only considered when the mode is "manual")
- * @return {Promise<AxiosResponse<any>>}
+ * @return {Promise<any>} (AxiosResponse<any>)
```

Comment:
To use types from packages in JSDocs we may use import.

```suggestion
 * @return {Promise<import('axios').AxiosResponse<any>>}
```

Reply:
We don't have `axios` as a dependency in spreed repository, only inside `@nextcloud/axios` bundle. But i'm not sure we could import a type from there, only if it was re-exported

Reply:
> We don't have `axios` as a dependency in spreed repository, only inside `@nextcloud/axios` bundle. But i'm not sure we could import a type from there, only if it was re-exported

`axios` is a sub-dependency of `@nextcloud/axios`, it is not bundled to `@nextcloud/axios` build. So it is installed as well as ESLint or Webpack, for example.

This typing should work.